app/components/account-reset-password-form/template.hbs

```diff
@@ -30,7 +30,7 @@
       value=changeset.password
       errors=changeset.error.password.validation
       submitted=form.status.tried
-      clue='must have at least 8 characters and one number'
+      clue=(if changeset.error.password 'must be at least 8 characters and 1 number' '')
```

Comment:
you don't need that empty string for the else case, it can just be `(if <condition> 'some string')` if the other case is null